editor/src/components/canvas/controls/comment-popup.tsx

```diff
@@ -75,10 +81,10 @@ function CommentThread({ x, y }: CommentThreadProps) {
       // Create a new thread
       createThread({
         body,
-        metadata: { type: 'canvas', x: x, y: y, name: name ?? 'Anonymous' },
+        metadata: { type: 'canvas', x: x, y: y, name: name ?? 'Anonymous', colorIndex: colorIndex },
```

Comment:
can `name` be picked from the presence as well?

Reply:
I would like to pick the name from resolveUsers, so this will be removed. That is what can guarantee that the name here and in the comment popup is the same

Reply:
fine for me! maybe worth adding a TODO comment for this